statscache_plugins/releng/__init__.py

```diff
@@ -164,12 +166,13 @@ def initialize(self, session):
             except Exception as e:
                 log.exception(
                     "Error during initializing releng plugin '{}': {}".format(
-                        plugin.idx, e), exc_info=True)
+                        plugin.ident, e), exc_info=True)
 
     def cleanup(self):
         pass
 
     def load_plugins(self, config, model):
+        # TODO: plugins for a plugin? This shoudln't be necessary
```

Comment:
Oh whoops, I totally forgot I left that in there! Didn't mean to be so critical :) I'd say there's potential for a "plugin manager" class to fill this sort of purpose